src/main/fc/stats.c

```diff
@@ -56,29 +60,38 @@ void statsInit(void)
     dispatchEnable();
 }
 
+void writeStats(struct dispatchEntry_s* self);
+
+dispatchEntry_t writeStatsEntry =
+{
+    writeStats, 0, NULL, false
+};
+
```

Comment:
```suggestion
static void writeStats(dispatchEntry_t *self);

static dispatchEntry_t writeStatsEntry = { writeStats, 0, NULL, false };
```
This function declaration and variable definition can go to the top, for example in line 51. And because they are "private", they can be made `static`.

Reply:
done